src/gsi_ncdiag/gsi_ncdiag.py

```diff
@@ -181,11 +181,13 @@
 
 gsi_add_vars_allsky = {
     'Observation_Type': 'ObsType',
+    'Observation_Subtype': 'ObsSubType',
     'Prep_Use_Flag': 'PreUseFlag',
     'Analysis_Use_Flag': 'GsiUseFlag',
     'Nonlinear_QC_Rel_Wgt': 'GsiQCWeight',
-    'Errinv_Adjust': 'GsiAdjustObsError',
```

Comment:
Isn't this the 'default' GSI name though? Should we keep support for inverse error and input error?